docs/topics/command-line.md

```diff
@@ -2,10 +2,13 @@
 
 Every Kotlin release ships with a standalone version of the compiler. You can download the latest version manually or via a package manager.
 
->Installing the command-line compiler is not an essential step to use Kotlin. A general way to write Kotlin applications is using an
->IDE - [IntelliJ IDEA](https://www.jetbrains.com/idea/) or [Android Studio](https://developer.android.com/studio).
->They provide full Kotlin support out of the box without needing additional components. Learn how to 
-[get started with Kotlin in an IDE](getting-started.md).
+> Installing the command-line compiler is not an essential step to use Kotlin.
+> The common approach is to write Kotlin applications using IDEs or code editors with official Kotlin support,
+> such as [IntelliJ IDEA](https://www.jetbrains.com/idea/), [JetBrains Fleet](https://www.jetbrains.com/fleet/),
+> or [Android Studio](https://developer.android.com/studio).
+> These provide full Kotlin support right out of the box, no extra components needed.
```

Comment:
```suggestion
> They provide full Kotlin support right out of the box, no extra components needed.
```

I feel that "they" is a clearer word here, "these" more naturally refers to something perceived visually.